source/Calamari.Common/Calamari.Common.csproj

```diff
@@ -58,14 +58,15 @@
     </ItemGroup>
 
     <ItemGroup>
-      <PackageReference Include="JavaPropertiesParser" Version="0.2.1" />
-      <PackageReference Include="Microsoft.Web.Xdt" Version="3.1.0" />
-      <PackageReference Include="Octopus.Versioning" Version="4.3.5" />
-      <PackageReference Include="Octostache" Version="2.14.0" />
-      <PackageReference Include="SharpCompress" Version="0.24.0" />
-      <PackageReference Include="XPath2" Version="1.0.12" />
-      <PackageReference Include="YamlDotNet" Version="8.1.2" />
-      <PackageReference Include="System.ValueTuple" Version="4.4.0" />
+        <PackageReference Include="JavaPropertiesParser" Version="0.2.1" />
+        <PackageReference Include="Microsoft.Web.Xdt" Version="3.1.0" />
+        <PackageReference Include="Octopus.Versioning" Version="5.0.5" /> <!-- TODO @team-deploy-fnm: Octopus.Versioning > 5.0.5 is not compatible with net40. This package should be updated when Calamari moves to net5.0 -->
+        <PackageReference Include="Octopus.CoreUtilities" Version="2.1.449" />
```

Comment:
`CoreUtilities` was removed as a dependency from Calamari, so it needs to be specified as a dependency here since quite a few files rely on it in all 3 projects.